### DISCUSSION TITLE
Rotate is not available

### DISCUSSION THREAD

**Discussion by arvigeus** [tailwindlabs/tailwindcss#9163] [Help] [answered]
I am trying to do expanding TreeView with [rotate](https://tailwindcss.com/docs/rotate). For some reason `.rotate-90` definition is missing from styles.

Here is an [example](https://codesandbox.io/s/heuristic-kowalevski-2h7ccb?file=/app/modules/FileTreeView/components/ItemArrow.tsx) (the arrow in front of directories).

Am I missing something, or this is a bug?

**Comment by wongjn** [comment, marked as answer]
Seems like tailwind isn't actually installed and so perhaps the generated CSS is outdated. This is what I see in the example:

```
[0] > dev:css
[0] > tailwindcss -w -i ./styles/app.css -o app/styles/app.css
[0]
[0] sh: 1: tailwindcss: not found
[0] npm run dev:css exited with code 127
```

**Reply by arvigeus** [reply]
No idea how it got there, since it was working before. Anyway, thanks!
